src/more-infos/more-info-content.html

```diff
@@ -35,11 +35,14 @@
   },
 
   stateObjChanged: function (stateObj) {
-    if (!stateObj) return;
-
-    window.hassUtil.dynamicContentUpdater(
-      this, 'MORE-INFO-' + window.hassUtil.stateMoreInfoType(stateObj).toUpperCase(),
-      { hass: this.hass, stateObj: stateObj });
+    if (!stateObj) {
+      window.hassUtil.dynamicContentUpdater(
+        this, null, { isVisible: false });
```

Comment:
Before my change closing more-info would just hide the element. At least in chrome camera stream  would still continue to run.

I tried to remove (without resetting the URL) the more-info-camera from the DOM, but the stream would still run.

I refactored this to use different functions for the null and non-null cases.

Regarding speed:
1) opening more-info-light [no change]
2) closing it. [sets isVisible to false, light don't care but a small amount of work is done]
3) opening it again [exactly same effort. The dom is still there and received a stateObj like before my change]